src/nvim/ex_cmds.lua

```diff
@@ -2158,7 +2158,7 @@ return {
   },
   {
     command='resize',
-    flags=bit.bor(RANGE, NOTADR, TRLBAR, WORD1),
+    flags=bit.bor(RANGE, NOTADR, TRLBAR, WORD1, CMDWIN),
```

Comment:
would be worth updating the comment next to the declaration of `CMDWIN` in `src/nvim/ex_cmds_defs.h` as in the Vim patch.